custom/sensors/CustomCANCoder.java

```diff
@@ -82,7 +84,8 @@ public boolean getDirection() {
 
     @Override
     public boolean getDirectionSafely() throws InvalidSensorException {
-        return !reverseDirection == (getPosition() >= 0);
+        // getPosition() returns status signal, can't use >= on status signal
+        return !reverseDirection == (getPosition().getValue() >= 0);
```

Comment:
i think you can do getPosition().getValue() and then change what the function is returning to smth else and it works? i don't remember but it works in a different file (idk which one)